test/integration/smoke/test_network.py

```diff
@@ -2018,12 +2024,14 @@ def test_02_verifyRouterIpAfterNetworkRestart(self):
 
     @attr(tags=["advanced", "shared"])
     def test_03_destroySharedNetwork(self):
+        if self.hv.lower() == 'simulator':
+            self.skipTest("Hypervisor is simulator - skipping Test..")
         routerIp = self.services["shared_network"]["routerip"]
         try:
             self.shared_network.delete(self.apiclient)
         except Exception as e:
             self.fail("Failed to destroy the shared network")
-
+        self._cleanup.remove(self.shared_network)
```

Comment:
this should be done in the teardown method,

Reply:
The intention of removing the shared network from the cleanup list is that, if the deletion of the n/w happens as part of the test, then it need not be cleaned up during the teardown phase, else, if for any reason this test fails or is skipped, the network will be deleted during teardown